### PR TITLE
relax timeouts for some service protocol calls

### PR DESCRIPTION
**Pull request by devoncarew**
- relax timeouts for some service protocol calls

@yjbanov, I'm seeing timeouts when debugging flutter apps - breakpoints introduce delays at unexpected places and trigger these timeouts. We may want to re-think the timeouts a bit.

### CODE REVIEW

File: packages/flutter_tools/lib/src/vmservice.dart [hunk 1]
```diff
@@ -557,7 +557,7 @@ class VM extends ServiceObjectOwner {
     }
   }
 
-  // Note that this function does not reload the isolate if it found
+  // Note that this function does not reload the isolate if it's found
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/vmservice.dart]: nit: remove "note that"

Comment (devoncarew) [inline packages/flutter_tools/lib/src/vmservice.dart]: done!

**Review by yjbanov — APPROVED** [review]

**Comment by yjbanov** [comment]
I believe there should be an issue filed about it. In short, yes, if the debugger is running the timeouts should be turned off.

**Comment by yjbanov** [comment]
Couldn't find it. Filed https://github.com/flutter/flutter/issues/7470.

**Comment by johnmccutchan** [comment]
I'm not a fan of putting policy in library code. The caller of the vm service rpc should decide what an appropriate timeout should be.

**Pull request merged**